WireAPI/Tests/WireAPITests/APIs/UpdateEventsAPI/__Snapshots__/UpdateEventsAPITests/testGetUpdateEvents.request-1-v6.txt

```diff
@@ -1,4 +1,2 @@
-- HTTPRequest
-  - path: "/v6/notifications?client=abcd1234&since=2eeeb5e4-df85-4aef-9eb2-289981f086ab&size=500"
-  - method: get
-  - body: none
+curl \
```

Comment:
is that valid curl at the end if it does not contain a baseURL?

Reply:
I guess not, but it's because the snapshot is taken before the request is resolved against the base url.